app/templates/contact/confirmation.html

```diff
@@ -30,6 +30,59 @@
 
 
 {% block content %}
+  <script nonce="{{ csp_nonce() }}">
+    document.addEventListener("DOMContentLoaded", function () {
+      const form = document.querySelector(".confirmation-email-form");
+      if (!form) return;
+
+      form.addEventListener("submit", async function (e) {
+        e.preventDefault();
+
+        const formData = new FormData(form);
+        const response = await fetch(window.location.href, {
+          method: "POST",
+          headers: {
+            "X-Requested-With": "XMLHttpRequest",
+          },
+          body: formData,
+        });
+
+        const content = await response.json();
+
+        if (response.ok && content.success) {
+          //Update placed in wrapper to avoid GTM reading this as a reload
+          const messageWrapper = document.createElement("div");
+          messageWrapper.innerHTML = `
+            <p class="govuk-body">
+              Your reference number was sent to <strong>${content.email}</strong>.
```

Comment:
Consider showing and hiding by updating class rather than dhtml so we can put translations in and not require server-side code in JS. Either way Accessibility will need checking.